tests/models/components/test_seq2seq_encoder.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import torch
+
+from pie_models.models.components.seq2seq_encoder import (
+    ACTIVATION_TYPE2CLASS,
+    RNN_TYPE2CLASS,
+    build_seq2seq_encoder,
+)
+
+
+def test_no_encoder():
+    seq2seq_dict = {}
+    encoder, _ = build_seq2seq_encoder(seq2seq_dict, 10)
+    assert encoder is None
+
+    seq2seq_dict = {
+        "type": "sequential",
+        "rnn_layer": {
+            "type": "none",
+        },
+    }
+    encoder, _ = build_seq2seq_encoder(seq2seq_dict, 10)
+    assert len(encoder) == 0
+
+
+@pytest.mark.parametrize("seq2seq_enc_type", list(RNN_TYPE2CLASS))
+def test_rnn_encoder(seq2seq_enc_type):
+    seq2seq_dict = {
+        "type": seq2seq_enc_type,
+        "hidden_size": 10,
+    }
+    encoder, _ = build_seq2seq_encoder(seq2seq_dict, 10)
+    assert encoder is not None
+    assert isinstance(encoder.rnn, type(RNN_TYPE2CLASS[seq2seq_enc_type](10, 10)))
+
+
+@pytest.mark.parametrize("bidirectional", [True, False])
+def test_output_size(bidirectional):
+    seq2seq_dict = {
+        "type": "lstm",
+        "bidirectional": bidirectional,
+        "hidden_size": 10,
+    }
+
+    input_size = 10
+    assert_output_size = input_size * 2 if bidirectional else input_size
+    _, output_size = build_seq2seq_encoder(seq2seq_dict, input_size)
+    assert output_size is not None
+    assert output_size == assert_output_size
```

Comment:
Why just for the lstm? This test repeats the setup for lstm with bidirectional=False from `test_rnn_encoder`. And it has a strange name considering what's in fact tested. I would propose to integrate that into `test_rnn_encoder`.